source/tools/timemory-avail/CMakeLists.txt

```diff
@@ -7,8 +7,15 @@ endif()
 add_executable(timemory-avail ${_EXCLUDE}
     ${CMAKE_CURRENT_LIST_DIR}/timemory-avail.cpp
     ${CMAKE_CURRENT_LIST_DIR}/timemory-avail.hpp)
+set_target_properties(timemory-avail PROPERTIES DEBUG_POSTFIX "${CMAKE_DEBUG_POSTFIX}")
 target_include_directories(timemory-avail PRIVATE ${CMAKE_CURRENT_LIST_DIR})
 target_link_libraries(timemory-avail PRIVATE timemory-extensions timemory-headers)
+
+if (NOT TIMEMORY_BUILD_CEREAL)
+    # Shouldn't timemory-headers already have a dependency on timemory::timemory-compile-options
+    target_link_libraries(timemory-avail PRIVATE timemory-compile-options)
+endif()
```

Comment:
You can remove this after the change to `target_include_directories(timemory-cereal INTERFACE ${TIMEMORY_CEREAL_INCLUDE})` as `timemory-headers` will provide `timemory-cereal` and `timemory-cereal` will provide the include path, wherever it is.

Reply:
I am just going to accept this since it will be removed soon in #101